public/js/wSlider.js

```diff
@@ -20,13 +21,41 @@ $.widget('custom.SLIDER', {
       orientation: this.options.format.orientation,
       stop: this._triggerSocketEvent.bind(this)
     })
+    this.currentValue = this.options.format.default
     this.element.children('.widget-content').html(sliderElement)
+    // let's call it once on init.
```

Comment:
I can't tell if lines 26 and 28 are TODOs are a description of what's happening.
If the former, start them with
// TODO:
for consistency. 
I suspect line 28 is a comment about line 29. If so, it's redundant detail.

Reply:
I scaffold a lot of my actions with comments, removed.